public/projects/emhoracek/smooch.mdx

```diff
@@ -0,0 +1,35 @@
+---
+name: smooch
+repoUrl: https://github.com/emhoracek/smooch
+description: An implementation of the Kisekae Set System for the web
+maintainer: emhoracek
+created: 2022-10-04T23:05:00.147Z
+languages:
```

Comment:
**suggestion:** seems like Haskell should be listed too, right?
```suggestion
languages:
  - Haskell
```

Reply:
Looks good to me!